src/__tests__/functions.test.js

```diff
@@ -0,0 +1,13 @@
+import { fahreinheitToCelcius, getHoursAndMinutes } from '../utils/functions';
+
+describe('Function tests', () => {
+  it('should be equal to ', () => {
+    const date = getHoursAndMinutes(1673041519);
+    expect(date).toEqual('09:45 PM');
```

Comment:
- [ ] I tested in my time zone UTC -5 (20:09), but I still having the same result, as the last code reviewer said, please 🙏🏽 fix this test to pass all the tests

![image](https://user-images.githubusercontent.com/58642949/211124335-ae7807f1-061a-4322-9df3-3637903420fa.png)

![image](https://user-images.githubusercontent.com/58642949/211124355-9eb550b4-79df-4dc4-a90c-22e4211a5944.png)